rites/roman1969/src/locales/fr.ts

```diff
@@ -152,19 +152,24 @@ export const locale: Locale = {
   },
 
   names: {
+    achard_of_saint_victor_bishop: 'Bienheureux Achard, évêque d’Avranches († 1171)', // src: mr_fr_1982_ed2_coutances
     adalbert_of_prague_bishop: 'Saint Adalbert, évêque de Prague et martyr († 997)',
     adalbert_of_prague_bishop_patron_of_poland:
       'Saint Adalbert, évêque de Prague, martyr et patron de la Pologne († 997)',
     adelaide_of_burgundy_empress: 'Sainte Adélaîde, impératrice († 999)',
+    adeline_de_mortain_abbess_and_the_saints_of_savigny:
+      'Sainte Adeline de Mortain († v. 1125) et les saints de Savigny', // src: mr_fr_1982_ed2_coutances
     adelphus_of_metz_bishop: 'Saint Adelphe, évêque († Vème s.)',
     agatha_of_sicily_virgin: 'Sainte Agathe, vierge et martyre en Sicile († 251)',
     agnes_of_rome_virgin: 'Sainte Agnès, vierge et martyre († v. 304)',
     albert_the_great_bishop:
       'Saint Albert le Grand, frère prêcheur, évêque de Ratisbonne, docteur de l’Église († 1280)',
     all_holy_bishops_of_the_archdiocese_of_lyon: 'Tous les Saints Évêques du diocèse de Lyon',
+    all_holy_bishops_of_the_diocese_of_coutances: 'Tous les Saints Évêques du diocèse de Coutances', // src: mr_fr_1982_ed2_coutances
     all_saints: 'Tous les Saints',
     all_saints_of_the_archdiocese_of_lyon: 'Tous les Saints du diocèse de Lyon', // src: mr_fr_2014_ed2_lyon
     all_saints_of_the_archdiocese_of_paris: 'Tous les Saints du diocèse de Paris',
+    all_saints_of_the_diocese_of_coutances: 'Tous les saints du diocèse de Coutances et d’Avranches', // src: mr_fr_1982_ed2_coutances
```

Comment:
![Screenshot 2024-10-11 at 11 23 58](https://github.com/user-attachments/assets/b84d3f8d-3993-4abf-88e6-65c75a635d6f)

Reply:
Regarding `Tous`/`diocése`: the Missal has some inconsistencies. This celebration includes `Tous`/`diocése`, but `all_holy_bishops_of_the_diocese_of_coutances` does not. I think you should choose which wording we will use (probably keeping `Tous`/`diocése`, as it makes more sense to me) and stick with it (as you already do :wink:).

Is it grammatically correct to omit `d’` before `Avranches`? I get it why it could be correct, however, in romcal we currently include the preposition. :thinking: If yes, remove `d’`.

Reply:
Yes, the names in the diocesan propers are not always very consistent with each other. Maybe they didn’t add `Tous`/`diocèse` to avoid an overly long name?  
In such cases, I prefer to follow the example of the French missal, which is much more consistent. So here, I will add `Tous`/`diocèse`.

As for the `d’` or its omission before `Avranche`, both are correct. So I can leave it out here.